packages/neon/neon/lib/src/router.dart

```diff
@@ -176,12 +196,22 @@ class LoginRoute extends GoRouteData {
   }
 }
 
+/// {@template AppRoutes.LoginFlowRoute}
+/// Route for the the [LoginFlowPage].
+///
+/// Redirects to [_AddAccountFlowRoute] when at least one account is already
```

Comment:
Does it work link to a private class?

Reply:
It does work in the code and the generated docs have this highlighted.
There won't be any link generated as private members documentation is not included in the generated html.

As this entire file isn't exported for public use it will not be included in the generated html in the first place.